docs/pages/reference/access-controls/authentication.mdx

```diff
@@ -23,12 +23,11 @@ possible values (types) of MFA:
   See our [Second Factor - WebAuthn](../../admin-guides/access-controls/guides/webauthn.mdx) guide for detailed
   instructions on setting up WebAuthn for Teleport.
 - `on` enables both TOTP and WebAuthn, and all local users are required to have at least one MFA device registered.
-- `optional` enables both TOTP and WebAuthn but makes it optional for users. Local users that register a MFA device will
-  be prompted for it during login. This option is useful when you need to gradually enable MFA usage before switching
-  the value to `on`.
-- `off` turns off multi-factor authentication.
 
 <Admonition type="note">
+
+  As of v16 `optional` and `off` are not supported as [multi-factor authentication is now required for local users](../../changelog.mdx#multi-factor-authentication-is-now-required-for-local-users).
```

Comment:
I'd remove this. We don't mention `optional` or `off` anywhere else in the document, so there's no need to go out of your way to discuss these options.